pkgs/applications/networking/sniffers/qtwirediff/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qtbase
+, qmake
+, qtwayland
+, wrapQtAppsHook
+}:
+
+
+stdenv.mkDerivation {
+  pname = "qtwirediff";
+  version = "unstable-2022-01-15";
+
+  src = fetchFromGitHub {
+    owner = "aaptel";
+    repo = "qtwirediff";
+    rev = "64e163e0bfaf1e6d33379de153af53820c146411";
+    hash = "sha256-tEDUEMRiYyCFJYmD6EB+2ZocCAprDrmmJLX0Aa76djU=";
+  };
+
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qtwayland
+  ];
+
+  postInstall = ''
```

This seems to be exactly the installPhase though, so maybe
```suggestion
  installPhase = ''
    runHook preInstall
```

---

Sounds good, but I will have to look at what the preInstall hook does.

---

Just tested also works without explicitly calling the hook, is there a reason to do it explicitly?

---

addressed below